openapi/p4pa-workflow-hub.openapi.yaml

```
@@ -1,31 +1,62 @@
-openapi: 3.1.0
+openapi: 3.0.1
 info:
   title: p4pa-workflow-hub
   description: Api and Models
-  version: "1.0.0"
+  version: 0.0.1
+servers:
+  - url: "http://localhost:8080/workflowhub"
 paths:
-  /api/v1/greet:
+  /{workflowId}/status:
     get:
-      summary: "Hello World"
-      description: "Sample endpoint that return greetings"
+      tags:
+        - WorkflowHub
```

Let's give the entity name as tag, not the application itself
```suggestion
        - Workflow
```